plasma_framework/test/endToEndTests/FastExit.e2e.test.js

```diff
@@ -0,0 +1,265 @@
+            const tranferTxBlockNum = (await this.framework.nextChildBlock()).toNumber();
+            this.transferUtxoPos = buildUtxoPos(tranferTxBlockNum, 0, 0);
+
+            const outputLC = new PaymentTransactionOutput(
+                OUTPUT_TYPE_PAYMENT,
+                TRANSFER_AMOUNT,
+                this.liquidity.address,
+                ETH,
+            );
+            const outputAlice = new PaymentTransactionOutput(
+                OUTPUT_TYPE_PAYMENT,
+                DEPOSIT_VALUE - TRANSFER_AMOUNT,
+                alice,
+                ETH,
+            );
+            this.transferTxObject = new PaymentTransaction(1, [this.depositUtxoPos], [outputLC, outputAlice]);
+            this.transferTx = web3.utils.bytesToHex(this.transferTxObject.rlpEncoded());
+            this.merkleTreeForTransferTx = new MerkleTree([this.transferTx]);
+            this.merkleProofForTransferTx = this.merkleTreeForTransferTx.getInclusionProof(this.transferTx);
+
+            await this.framework.submitBlock(this.merkleTreeForTransferTx.root, { from: authority });
+        };
+
+        describe('Given contracts deployed, exit game and ETH vault registered', () => {
+            before(setupContracts);
+
+            it('should not allow to call processExit from outside of exit game controller contract', async () => {
+                await expectRevert(
+                    this.exitGame.processExit(0, config.registerKeys.vaultId.eth, constants.ZERO_ADDRESS),
+                    'Caller address is unauthorized.',
+                );
+            });
+
+            describe('Given Alice deposited ETH and transferred some value to the Liquidity Contract', () => {
+                before(async () => {
+                    this.aliceBalanceBeforeDeposit = new BN(await web3.eth.getBalance(alice));
+                    this.ethVaultBalanceBeforeDeposit = new BN(await web3.eth.getBalance(this.ethVault.address));
+                    const { receipt } = await aliceDepositsETH();
+                    this.aliceDepositReceipt = receipt;
+                    await aliceTransferSomeEthToLC();
+                });
+                it('should have deposited ETH from Alice to vault', async () => {
+                    const aliceBalanceAfterDeposit = new BN(await web3.eth.getBalance(alice));
+                    const ethVaultBalanceAfterDeposit = new BN(await web3.eth.getBalance(this.ethVault.address));
+                    const expectedAliceBalance = this.aliceBalanceBeforeDeposit
+                        .sub(new BN(DEPOSIT_VALUE))
+                        .sub(await spentOnGas(this.aliceDepositReceipt));
+                    const expectedEthVaultBalance = this.ethVaultBalanceBeforeDeposit.add(new BN(DEPOSIT_VALUE));
+
+                    expect(aliceBalanceAfterDeposit).to.be.bignumber.equal(expectedAliceBalance);
+                    expect(ethVaultBalanceAfterDeposit).to.be.bignumber.equal(expectedEthVaultBalance);
+                });
```

Comment:
I believe that we do not want to test it here - we should only test the code that we are actually testing and we own - we should not test Ethereum VM and also Plasma Framework - Plasma Framework in this case is a dependency and has its own test.

Reply:
true removed `c4c18de`